### PR TITLE
Adapted for changes in question handling

### PR DESCRIPTION
**Pull request by SandroGlueck**
*Added ItemType model
*Extended Item with item_type_id
*Added hint and tool tip for Question
*Added tooltip for AnswerOption
*Added dynamic ItemType in submission

### CODE REVIEW

File: src/app/review-item/components/question/question.component.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
-<h4>{{question?.content}}</h4>
+<h4>{{question?.content}}<span *ngIf="question?.hint"><mat-icon [matTooltip]="question?.hint">help_outline</mat-icon></span></h4>
 <div *ngIf="question?.info">
-  <p>{{question?.info}}</p>
+  <p [innerHTML]="question?.info"></p>
 </div>
 <form *ngIf="form" [formGroup]="form">
   <mat-radio-group formControlName="option">
-    <mat-radio-button *ngFor="let option of question?.options" [value]="option" [checked]="false">{{option.text}}</mat-radio-button>
+    <mat-radio-button *ngFor="let option of this.options" [value]="option" [checked]="false" [matTooltip]="option.tooltip">
```
Comment (PotSchmoell) [inline src/app/review-item/components/question/question.component.html:7]: Shouldn't we also have a small icon here to signalize that a tooltip exists?

File: src/app/submit-item/components/submit/submit.component.html [hunk 2]
```diff
@@ -18,8 +18,7 @@ <h1>Ein neuer Fall für das DetektivKollektiv? <span class="alt">Reiche ihn hier
             <mat-radio-group formControlName="typeFormControl">
               <mat-label>Bitte wähle einen Typ aus.</mat-label>
               <br>
-              <mat-radio-button value="claim">Textnachricht / Aussage</mat-radio-button>
-              <mat-radio-button value="article">Link zu Artikel</mat-radio-button>
+              <mat-radio-button *ngFor="let itemType of itemTypes" [value]="itemType.id">{{itemType.name | translate}}</mat-radio-button>
```
Comment (PotSchmoell) [inline src/app/submit-item/components/submit/submit.component.html:21]: I think it would be good to show a loading spinner until the options are loaded

**Review by PotSchmoell — CHANGES_REQUESTED** [review]

File: src/app/review-item/components/question/question.component.html (hunk 1 above)
Comment (SandroGlueck) [reply src/app/review-item/components/question/question.component.html:7]: Isn't this screen supposed to be changed entirely according to Daniels mocks?

Comment (PotSchmoell) [reply src/app/review-item/components/question/question.component.html:7]: Yeah, you're right. But the mocks also include the icon. :P 
Are you on the re-design already?

**Comment by SandroGlueck** [comment]
Added the loader - I'd suggest we keep the change in design in mind for the large scale redesign

**Review by PotSchmoell — APPROVED** [review]

**Pull request merged**